README.md

```diff
@@ -64,27 +69,35 @@ Describe the rules of scope in JavaScript.
 
 Your Answer:
 ```text
+Scope is a concept referring to what can be accessed from where in JS. If an item that can be named (things like functions or variables) are named without those modifiers and are outside of any function they are considered to be in the "global" scope which by default refers to the "window" in a browser. If functions or variables are called within a function, or if they use the keywords "var" or "function" then they are only accessible within THAT present scope and the ones outside of them. NOT the ones inside them. This is why hoisting works.
 ```
 
 ### Question #6
 
 Define an object and store it in a variable `pizza`. The object should have 2
 properties: a temperature (set to 70), and a method called `bake`. When called,
-this method should set the pizza's temperature to be 300. Note: you may not use
+this method should set the pizzas temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza = {
+  temperature = 70,
+  bake: function(){
+    this.temperature = 300;
+  }
+}
 ```
 
 ### Question #7
 
-Define a global variable instructor and set it equal to your Squad Instructor's Name. Then, define the same as a local variable instead.
+Define a global variable instructor and set it equal to your Squad Instructors Name. Then, define the same as a local variable instead.
 
 Your Answer:
 ```js
-// write code here
+// instructor = "Adrian";    /// That's the global one, see? No "var" :)
+// var instructor = "Adrian";   /// that variable creates a local scope and is not global.
+
 ```
```

Comment:
:+1: Although `var instructor` does not create a local scope. That just defines it in the current scope.

And technically, if you write out `var instructor` in the global scope, it is a (locally) global variable.